clients/client-accessanalyzer/README.md

```diff
@@ -3,4 +3,203 @@
+
+## Getting Started
+
+### Import
+
+The AWS SDK is modulized by clients and commands.
+To send a request, you only need to import the `AccessAnalyzerClient` and
+the commands you need, for example `CreateAnalyzerCommand`:
+
+```js
+// ES5 example
+const { AccessAnalyzerClient, CreateAnalyzerCommand } = require("@aws-sdk/client-accessanalyzer");
+```
+
+```ts
+// ES6+ example
+import { AccessAnalyzerClient, CreateAnalyzerCommand } from "@aws-sdk/client-accessanalyzer";
+```
+
+### Usage
+
+To send a request, you:
+
+- Initiate client with configuration (e.g. credentials, region).
+- Initiate command with input parameters.
+- Call `send` operation on client with command object as input.
+- If you are using a custom http handler, you may call `destroy()` to close open connections.
+
+```js
+// a client can be shared by difference commands.
+const client = new AccessAnalyzerClient({ region: "REGION" });
+
+const params = {
+  /** input parameters */
+};
+const command = new CreateAnalyzerCommand(params);
+```
+
+#### Async/await
+
+We recommend using [await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await)
+operator to wait for the promise returned by send operation as follows:
+
+```js
+// async/await.
+try {
+  const data = await client.send(command);
+  // process data.
+} catch (error) {
+  // error handling.
+} finally {
+  // finally.
+}
+```
+
+Async-await is clean, concise, intuitive, easy to debug and has better error handling
+as compared to using Promise chains or callbacks.
+
+#### Promises
+
+You can also use [Promise chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises#chaining)
+to execute send operation.
+
+```js
+client.send(command).then(
+  (data) => {
+    // process data.
+  },
+  (error) => {
+    // error handling.
+  }
+);
+```
+
+Promises can also be called using `.catch()` and `.finally()` as follows:
+
+```js
+client
+  .send(command)
+  .then((data) => {
+    // process data.
+  })
+  .catch((error) => {
+    // error handling.
+  })
+  .finally(() => {
+    // finally.
+  });
+```
+
+#### Callbacks
+
+We do not recommend using callbacks because of [callback hell](http://callbackhell.com/),
+but they are supported by the send operation.
+
+```js
+// callbacks.
+client.send(command, (err, data) => {
+  // proccess err and data.
+});
+```
+
+#### v2 compatible style
+
+The client can also send requests using v2 compatible style as follows:
+
+```ts
+import * as AWS from "@aws-sdk/client-accessanalyzer";
+const client = new AWS.AccessAnalyzer({ region: "REGION" });
+
+// async/await.
+try {
+  const data = client.createAnalyzer(params);
+  // process data.
+} catch (error) {
+  // error handling.
+}
+
+// Promises.
+client
+  .createAnalyzer(params)
+  .then((data) => {
+    // process data.
+  })
+  .catch((error) => {
+    // error handling.
+  });
+
+// callbacks.
+client.createAnalyzer(params, (err, data) => {
+  // proccess err and data.
+});
+```
+
```

Comment:
Maybe add a line here stating that 
```console
This style may results to bigger bundle size, and may be dropped in next major version.
```

Reply:
Added in [`6967168` (#1907)](https://github.com/aws/aws-sdk-js-v3/pull/1907/commits/6967168e626185af460d1c7a75d235ea42b8ce95)